src/main/java/com/ericsson/ei/notifications/HttpRequest.java

```diff
@@ -87,7 +87,7 @@ HttpRequest createHttpRequest() {
      * @return response     A boolean value of the request response
      * @throws AuthenticationException
      */
-    public boolean perform() throws AuthenticationException {
+    public boolean perform() throws Exception {
```

Comment:
Isn't it better to be specific and not throw generic exceptions?

Reply:
I was just lazy and made in generic since Eclipse wouldn't recognize the exception when I re-threw it in the catch block. I can list the ones I know.

Reply:
I now list the exceptions but in the InformSubscriber makeHTTPRequests function is still catch Exception since it's the same catch block for all errors.